TFP 1.46.0 data source: ibm_is_backup_policies, expected 'the empty list', but returned 'no BackupPolicies found'

<!---
Please note the following potential times when an issue might be in Terraform core:

* [Configuration Language](https://www.terraform.io/docs/configuration/index.html) or resource ordering issues
* [State](https://www.terraform.io/docs/state/index.html) and [State Backend](https://www.terraform.io/docs/backends/index.html) issues
* [Provisioner](https://www.terraform.io/docs/provisioners/index.html) issues
* [Registry](https://registry.terraform.io/) issues
* Spans resources across multiple providers

If you are running into one of these scenarios, we recommend opening an issue in the [Terraform core repository](https://github.com/hashicorp/terraform/) instead.
--->

<!--- Please keep this note for the community --->

### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

<!--- Thank you for keeping this note for the community --->

### Terraform CLI and Terraform IBM Provider Version

<!--- Please run `terraform -v` to show the Terraform core version and provider version(s). If you are not running the latest version of Terraform or the provider, please upgrade because your issue may have already been fixed. [Terraform documentation on provider versioning](https://www.terraform.io/docs/configuration/providers.html#provider-versions). --->
1.46.0
### Affected Resource(s)

<!--- Please list the affected resources and data sources. --->

* data ibm_is_backup_policies

### Terraform Configuration Files

<!--- Information about code formatting: https://help.github.com/articles/basic-writing-and-formatting-syntax/#quoting-code --->

Please include all Terraform configurations required to reproduce the bug. Bug reports without a functional reproduction may be closed without investigation.

```hcl
provider "ibm" {
  region = var.region
}

data ibm_is_backup_policies "available" {}

```

### Debug Output

<!---
Please provide a link to a GitHub Gist containing the complete debug output. Please do NOT paste the debug output in the issue; just paste a link to the Gist.

To obtain the debug output, see the [Terraform documentation on debugging](https://www.terraform.io/docs/internals/debugging.html).
--->
```
2022-10-05T13:08:07.006Z [INFO]  provider.terraform-provider-ibm_v1.46.0: 2022/10/05 13:08:07 [DEBUG] GET https://us-south.iaas.cloud.ibm.com/v1/backup_policies?generation=2&version=2022-09-13: timestamp=2022-10-05T13:08:07.006Z
2022-10-05T13:08:08.949Z [INFO]  provider.terraform-provider-ibm_v1.46.0: 2022/10/05 13:08:08 [Debug] Response:
HTTP/2.0 200 OK
Cache-Control: max-age=0, no-cache, no-store, must-revalidate
Cf-Cache-Status: DYNAMIC
Cf-Ray: 755662589a18aa15-DFW
Content-Type: application/json; charset=utf-8
Date: Wed, 05 Oct 2022 13:08:08 GMT
Expires: -1
Pragma: no-cache
Server: cloudflare
Strict-Transport-Security: max-age=31536000; includeSubDomains
Vary: Accept-Encoding
X-Content-Type-Options: nosniff
X-Request-Id: ee6057ef-e23f-4bf0-aa96-4f266002993a
X-Xss-Protection: 1; mode=block

{"backup_policies":[],"first":{"href":"https://us-south.iaas.cloud.ibm.com/v1/backup_policies?limit=50"},"limit":50,"total_count":0}: timestamp=2022-10-05T13:08:08.949Z
2022-10-05T13:08:08.951Z [ERROR] vertex "data.ibm_is_backup_policies.available" error: [ERROR] no BackupPolicies found
2022-10-05T13:08:08.951Z [ERROR] vertex "data.ibm_is_backup_policies.available" error: [ERROR] no BackupPolicies found
2022-10-05T13:08:08.951Z [ERROR] vertex "data.ibm_is_backup_policies.available (expand)" error: [ERROR] no BackupPolicies found

Error: [ERROR] no BackupPolicies found

  with data.ibm_is_backup_policies.available,
  on main.tf line 6, in data "ibm_is_backup_policies" "available":
   6: data ibm_is_backup_policies "available" {} # 
```
### Panic Output

<!--- If Terraform produced a panic, please provide a link to a GitHub Gist containing the output of the `crash.log`. --->

### Expected Behavior

<!--- What should have happened? --->

### Actual Behavior

<!--- What actually happened? --->
```
Error: [ERROR] no BackupPolicies found

  with data.ibm_is_backup_policies.available,
  on main.tf line 6, in data "ibm_is_backup_policies" "available":
   6: data ibm_is_backup_policies "available" {} # 
```

### Steps to Reproduce

<!--- Please list the steps required to reproduce the issue. --->

1. `terraform apply`

### Important Factoids

<!--- Are there anything atypical about your accounts that we should know? For example: Running in EC2 Classic? --->

```hcl
# This file is maintained automatically by "terraform init".
# Manual edits may be lost in future updates.

provider "registry.terraform.io/ibm-cloud/ibm" {
  version = "1.46.0"
  hashes = [
    "h1:j0+7z8mKyTIVrn/1HjiHmJdfHRoFcacyXg3kFacbTaY=",
    "zh:50b4360b3e0046e08f2fb6d486bb5a00575c0ffa297e4e9371e9cdd31abdcec3",
    "zh:541464d7588912d4e02170d06b6e3fa82da97dcadd04662c98291deb8bdc5ced",
    "zh:684fc6fd3b948bb49235862bb3b4f57aee81c30fcc23c53f08296282a761c93b",
    "zh:84f39c0ce3fe239d8b4e311f219a78834c9f619fc185cf3a3ca4910e089788c0",
    "zh:ca4cfacba214f7846dd5f2b4b86ff9067e18a50156e4c3d75de2a0b7905c5594",
    "zh:d63d9882d191fe2fd1bf6678bdf338812cf854b7cc7157869166dc5d1301d495",
  ]
}
```

### References

<!---
Information about referencing Github Issues: https://help.github.com/articles/basic-writing-and-formatting-syntax/#referencing-issues-and-pull-requests

Are there any other GitHub issues (open or closed) or pull requests that should be linked here? Vendor documentation? For example:
--->

* #0000

--- a/ibm/service/vpc/data_source_ibm_is_backup_policies.go
+++ b/ibm/service/vpc/data_source_ibm_is_backup_policies.go
@@ -226,9 +226,6 @@ func dataSourceIBMIsBackupPoliciesRead(context context.Context, d *schema.Resour
 		}
 
 	}
-	if len(matchBackupPolicies) == 0 {
-		return diag.FromErr(fmt.Errorf("[ERROR] no BackupPolicies found"))
-	}
 
 	d.SetId(dataSourceIBMIsBackupPoliciesID(d))
 

